automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/apps/CustomAppCreationSpec.scala

```diff
@@ -0,0 +1,95 @@
+package org.broadinstitute.dsde.workbench.leonardo
+package apps
+
+import org.broadinstitute.dsde.workbench.DoneCheckable
+import org.broadinstitute.dsde.workbench.google2.{streamFUntilDone, streamUntilDoneOrTimeout}
+import org.broadinstitute.dsde.workbench.leonardo.LeonardoApiClient.defaultCreateAppRequest
+import org.broadinstitute.dsde.workbench.leonardo.http.{ListAppResponse, PersistentDiskRequest}
+import org.http4s.{AuthScheme, Credentials, Uri}
+import org.broadinstitute.dsde.workbench.leonardo.LeonardoApiClient._
+import org.http4s.headers.Authorization
+import org.scalatest.{DoNotDiscover, ParallelTestExecution}
+
+import scala.concurrent.duration._
+
+@DoNotDiscover
+class CustomAppCreationSpec
+    extends GPAllocFixtureSpec
+    with LeonardoTestUtils
+    with GPAllocUtils
+    with ParallelTestExecution {
+  implicit val auth: Authorization =
+    Authorization(Credentials.Token(AuthScheme.Bearer, ronCreds.makeAuthToken().value))
+
+  "create and delete a custom app" in { _ =>
+    withNewProject { googleProject =>
```

Comment:
Currently all the app tests run in separate projects.. wondering if (in the future, not necessarily in this PR) we could re-use the same project + kubernetes cluster for >1 test case. This would speed up the tests, and we don't necessarily need to test k8s cluster creation for each app test. This may be more possible than before because we no longer delete the cluster+nodepool when we delete an app.